### PR TITLE
Add twohands symbols layout, fix twohands capitalization issues

### PR DESCRIPTION
**Pull request by Kommynct**
I added the two-handed symbols layout, pretty straightforward

this includes the changes made [here](https://github.com/dessalines/thumb-key/pull/548) and I don't know why because I made it a separate branch but I don't think it matters, likely need to close 548 in favor of this one.

There's also a numbered variant, which is mostly the same but with some numbers added so I don't have to use the # button.

### CODE REVIEW

File: hs_err_pid74296.log
Comment (dessalines) [inline hs_err_pid74296.log:1]: Remove this file from the PR.

**Review by dessalines — COMMENTED** [review]
Other than that, it looks good, thx!

File: hs_err_pid74296.log
Comment (Kommynct) [reply hs_err_pid74296.log:1]: Don't even know where that came from, fixed

**Review by dessalines — APPROVED** [review]

**Comment by dessalines** [comment]
Thx!

**Pull request merged**